### PR TITLE
plugdata: add livecheck

### PR DESCRIPTION
**Pull request by chenrui333**
Created by `brew bump`

---

Created with `brew bump-cask-pr`.

### CODE REVIEW

**Review by bevanjkay — DISMISSED** [review]

File: Casks/p/plugdata.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 cask "plugdata" do
-  version "0.8.2"
-  sha256 "9507f316d36593f5ca13b15eda26640279b71dd2bd383030d0a18b4fa9120864"
+  version "0.8.3-test"
```
Comment (bevanjkay) [inline Casks/p/plugdata.rb:2]: We should probably instead add a regex to the livecheck to skip tags with "test" in them.

**Review by bevanjkay — CHANGES_REQUESTED** [review]

File: Casks/p/plugdata.rb (hunk 1 above)
Comment (chenrui333) [reply Casks/p/plugdata.rb:2]: agree

File: Casks/p/plugdata.rb [hunk 2]
```diff
@@ -7,6 +7,11 @@
   desc "Plugin wrapper for PureData"
   homepage "https://github.com/timothyschoen/PlugData"
 
+  livecheck do
+    url :stable
```
Comment (bevanjkay) [inline Casks/p/plugdata.rb:11]: I think we have to use 

```suggestion
    url :url
```

**Review by bevanjkay — APPROVED** [review]

File: Casks/p/plugdata.rb (hunk 2 above)
Comment (chenrui333) [reply Casks/p/plugdata.rb]: 👍

**Pull request merged**